docs/csharp/language-reference/keywords/params.md

```diff
@@ -6,14 +6,16 @@ ms.date: 07/20/2015
 f1_keywords: 
   - "params_CSharpKeyword"
   - "params"
+  - "parameter array"
 helpviewer_keywords: 
   - "parameters [C#], params"
   - "params keyword [C#]"
+  - "parameter array"
 ms.assetid: 1690815e-b52b-4967-8380-5780aff08012
 ---
 # params (C# Reference)
 
-By using the `params` keyword, you can specify a [method parameter](method-parameters.md) that takes a variable number of arguments.
+By using the `params` keyword, you can specify an array of [method parameter](method-parameters.md) that takes a variable number of arguments.
```

Comment:
This sentence no longer makes sense. I would leave it as is and instead add "parameter array" to the `title` metadata.

Reply:
@gewarren Are you suggesting changing the title from "params keyword - C# Reference" to "params keyword parameter array- C# Reference"

Reply:
I'd suggest "params keyword for parameter arrays - C# reference".